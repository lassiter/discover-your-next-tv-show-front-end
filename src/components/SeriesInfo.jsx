import React, { Component } from 'react'
import { withStyles } from '@material-ui/core/styles';
import ExpansionPanel from '@material-ui/core/ExpansionPanel';
import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
import Typography from '@material-ui/core/Typography';
import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
import styled from 'styled-components'

// air_date: "2010-12-05"
// episode_count: 35
// id: 3627
// name: "Specials"
// overview: ""
// poster_path: "/kMTcwNRfFKCZ0O2OaBZS0nZ2AIe.jpg"
// season_number: 0

const PosterImage = styled.img`
  max-height: 150px;
  margin: auto 15px auto auto;
`
const Detail = styled.small`
  display: inline-block;
  width: 100%;
`
const InfoContainer = styled.div`
  max-width: 600px;
  margin: auto auto auto 15px;
`
const StyledPanel = styled(ExpansionPanel)`
  max-width: 800px;
`

export default class SeriesInfo extends Component {
  render() {
    console.log(this.props)
    const seasons = this.props.show.seasons.filter(season => season.name !== "Specials").map((season, index) => {
      return <Season key={index} season={season}/>
    })
    return (
      <div>
        {seasons}
      </div>
    )
  }
}

class Season extends Component {
  render() {
    let overview
    if (this.props.season.overview !== "") {
      overview = <p>{this.props.season.overview}</p>
    } else {
      overview = <></>
    }
    console.log(this.props)
    const posterImage = this.props.season.poster_path === null ? <></> : <PosterImage src={`https://image.tmdb.org/t/p/w500${this.props.season.poster_path}`}/>
    return (
      <StyledPanel>
        <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
          <h4>{this.props.season.name}</h4>
        </ExpansionPanelSummary>
        <ExpansionPanelDetails>
          {posterImage}
          <InfoContainer>
            {overview}
            <Detail>Episodes This Season: {this.props.season.episode_count}</Detail>
            <Detail>Air Date: {this.props.season.air_date}</Detail>
          </InfoContainer>

        </ExpansionPanelDetails>
      </StyledPanel>
    )
  }
}