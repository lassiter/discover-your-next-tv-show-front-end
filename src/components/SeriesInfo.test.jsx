import React from 'react';
import SeriesInfo from './SeriesInfo';
import renderer from 'react-test-renderer';
import jsonTestData from '../testJSON/got_sample_data.json'

it('renders correctly with data', () => {
  console.log(jsonTestData)
  const tree = renderer
    .create(<SeriesInfo show={JSON.parse(jsonTestData)}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});