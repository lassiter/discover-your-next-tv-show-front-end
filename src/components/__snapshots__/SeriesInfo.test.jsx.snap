// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with data 1`] = `
<div>
  <div
    className="MuiPaper-root-5 MuiPaper-elevation1-8 MuiPaper-rounded-6 MuiExpansionPanel-root-1 MuiExpansionPanel-rounded-2 sc-bxivhb lbdgCE"
  >
    <div
      aria-expanded={false}
      className="MuiButtonBase-root-38 MuiExpansionPanelSummary-root-32"
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex="0"
    >
      <div
        className="MuiExpansionPanelSummary-content-36"
      >
        <h4>
          Season 1
        </h4>
      </div>
      <div
        aria-hidden="true"
        className="MuiButtonBase-root-38 MuiIconButton-root-41 MuiExpansionPanelSummary-expandIcon-37"
        onBlur={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={-1}
      >
        <span
          className="MuiIconButton-label-46"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-47"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-61"
        />
      </div>
    </div>
    <div
      aria-hidden="true"
      className="MuiCollapse-container-56"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper-58"
      >
        <div
          className="MuiCollapse-wrapperInner-59"
        >
          <div
            className="MuiExpansionPanelDetails-root-60"
          >
            <img
              className="sc-bdVaJa gENAse"
              src="https://image.tmdb.org/t/p/w500/wgfKiqzuMrFIkU1M68DDDY8kGC1.jpg"
            />
            <div
              className="sc-htpNat gvzJoT"
            >
              <p>
                Trouble is brewing in the Seven Kingdoms of Westeros. For the driven inhabitants of this visionary world, control of Westeros' Iron Throne holds the lure of great power. But in a land where the seasons can last a lifetime, winter is coming...and beyond the Great Wall that protects them, an ancient evil has returned. In Season One, the story centers on three primary areas: the Stark and the Lannister families, whose designs on controlling the throne threaten a tenuous peace; the dragon princess Daenerys, heir to the former dynasty, who waits just over the Narrow Sea with her malevolent brother Viserys; and the Great Wall--a massive barrier of ice where a forgotten danger is stirring.
              </p>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Episodes This Season: 
                10
              </small>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Air Date: 
                2011-04-17
              </small>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiPaper-root-5 MuiPaper-elevation1-8 MuiPaper-rounded-6 MuiExpansionPanel-root-1 MuiExpansionPanel-rounded-2 sc-bxivhb lbdgCE"
  >
    <div
      aria-expanded={false}
      className="MuiButtonBase-root-38 MuiExpansionPanelSummary-root-32"
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex="0"
    >
      <div
        className="MuiExpansionPanelSummary-content-36"
      >
        <h4>
          Season 2
        </h4>
      </div>
      <div
        aria-hidden="true"
        className="MuiButtonBase-root-38 MuiIconButton-root-41 MuiExpansionPanelSummary-expandIcon-37"
        onBlur={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={-1}
      >
        <span
          className="MuiIconButton-label-46"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-47"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-61"
        />
      </div>
    </div>
    <div
      aria-hidden="true"
      className="MuiCollapse-container-56"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper-58"
      >
        <div
          className="MuiCollapse-wrapperInner-59"
        >
          <div
            className="MuiExpansionPanelDetails-root-60"
          >
            <img
              className="sc-bdVaJa gENAse"
              src="https://image.tmdb.org/t/p/w500/9xfNkPwDOqyeUvfNhs1XlWA0esP.jpg"
            />
            <div
              className="sc-htpNat gvzJoT"
            >
              <p>
                The cold winds of winter are rising in Westeros...war is coming...and five kings continue their savage quest for control of the all-powerful Iron Throne. With winter fast approaching, the coveted Iron Throne is occupied by the cruel Joffrey, counseled by his conniving mother Cersei and uncle Tyrion. But the Lannister hold on the Throne is under assault on many fronts. Meanwhile, a new leader is rising among the wildings outside the Great Wall, adding new perils for Jon Snow and the order of the Night's Watch.
              </p>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Episodes This Season: 
                10
              </small>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Air Date: 
                2012-04-01
              </small>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiPaper-root-5 MuiPaper-elevation1-8 MuiPaper-rounded-6 MuiExpansionPanel-root-1 MuiExpansionPanel-rounded-2 sc-bxivhb lbdgCE"
  >
    <div
      aria-expanded={false}
      className="MuiButtonBase-root-38 MuiExpansionPanelSummary-root-32"
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex="0"
    >
      <div
        className="MuiExpansionPanelSummary-content-36"
      >
        <h4>
          Season 3
        </h4>
      </div>
      <div
        aria-hidden="true"
        className="MuiButtonBase-root-38 MuiIconButton-root-41 MuiExpansionPanelSummary-expandIcon-37"
        onBlur={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={-1}
      >
        <span
          className="MuiIconButton-label-46"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-47"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-61"
        />
      </div>
    </div>
    <div
      aria-hidden="true"
      className="MuiCollapse-container-56"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper-58"
      >
        <div
          className="MuiCollapse-wrapperInner-59"
        >
          <div
            className="MuiExpansionPanelDetails-root-60"
          >
            <img
              className="sc-bdVaJa gENAse"
              src="https://image.tmdb.org/t/p/w500/5MkZjRnCKiIGn3bkXrXfndEzqOU.jpg"
            />
            <div
              className="sc-htpNat gvzJoT"
            >
              <p>
                Duplicity and treachery...nobility and honor...conquest and triumph...and, of course, dragons. In Season 3, family and loyalty are the overarching themes as many critical storylines from the first two seasons come to a brutal head. Meanwhile, the Lannisters maintain their hold on King's Landing, though stirrings in the North threaten to alter the balance of power; Robb Stark, King of the North, faces a major calamity as he tries to build on his victories; a massive army of wildlings led by Mance Rayder march for the Wall; and Daenerys Targaryen--reunited with her dragons--attempts to raise an army in her quest for the Iron Throne.
              </p>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Episodes This Season: 
                10
              </small>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Air Date: 
                2013-03-31
              </small>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiPaper-root-5 MuiPaper-elevation1-8 MuiPaper-rounded-6 MuiExpansionPanel-root-1 MuiExpansionPanel-rounded-2 sc-bxivhb lbdgCE"
  >
    <div
      aria-expanded={false}
      className="MuiButtonBase-root-38 MuiExpansionPanelSummary-root-32"
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex="0"
    >
      <div
        className="MuiExpansionPanelSummary-content-36"
      >
        <h4>
          Season 4
        </h4>
      </div>
      <div
        aria-hidden="true"
        className="MuiButtonBase-root-38 MuiIconButton-root-41 MuiExpansionPanelSummary-expandIcon-37"
        onBlur={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={-1}
      >
        <span
          className="MuiIconButton-label-46"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-47"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-61"
        />
      </div>
    </div>
    <div
      aria-hidden="true"
      className="MuiCollapse-container-56"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper-58"
      >
        <div
          className="MuiCollapse-wrapperInner-59"
        >
          <div
            className="MuiExpansionPanelDetails-root-60"
          >
            <img
              className="sc-bdVaJa gENAse"
              src="https://image.tmdb.org/t/p/w500/jXIMScXE4J4EVHUba1JgxZnWbo4.jpg"
            />
            <div
              className="sc-htpNat gvzJoT"
            >
              <p>
                The War of the Five Kings is drawing to a close, but new intrigues and plots are in motion, and the surviving factions must contend with enemies not only outside their ranks, but within.
              </p>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Episodes This Season: 
                10
              </small>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Air Date: 
                2014-04-06
              </small>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiPaper-root-5 MuiPaper-elevation1-8 MuiPaper-rounded-6 MuiExpansionPanel-root-1 MuiExpansionPanel-rounded-2 sc-bxivhb lbdgCE"
  >
    <div
      aria-expanded={false}
      className="MuiButtonBase-root-38 MuiExpansionPanelSummary-root-32"
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex="0"
    >
      <div
        className="MuiExpansionPanelSummary-content-36"
      >
        <h4>
          Season 5
        </h4>
      </div>
      <div
        aria-hidden="true"
        className="MuiButtonBase-root-38 MuiIconButton-root-41 MuiExpansionPanelSummary-expandIcon-37"
        onBlur={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={-1}
      >
        <span
          className="MuiIconButton-label-46"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-47"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-61"
        />
      </div>
    </div>
    <div
      aria-hidden="true"
      className="MuiCollapse-container-56"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper-58"
      >
        <div
          className="MuiCollapse-wrapperInner-59"
        >
          <div
            className="MuiExpansionPanelDetails-root-60"
          >
            <img
              className="sc-bdVaJa gENAse"
              src="https://image.tmdb.org/t/p/w500/7Q1Hy1AHxAzA2lsmzEMBvuWTX0x.jpg"
            />
            <div
              className="sc-htpNat gvzJoT"
            >
              <p>
                The War of the Five Kings, once thought to be drawing to a close, is instead entering a new and more chaotic phase. Westeros is on the brink of collapse, and many are seizing what they can while the realm implodes, like a corpse making a feast for crows.
              </p>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Episodes This Season: 
                10
              </small>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Air Date: 
                2015-04-12
              </small>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiPaper-root-5 MuiPaper-elevation1-8 MuiPaper-rounded-6 MuiExpansionPanel-root-1 MuiExpansionPanel-rounded-2 sc-bxivhb lbdgCE"
  >
    <div
      aria-expanded={false}
      className="MuiButtonBase-root-38 MuiExpansionPanelSummary-root-32"
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex="0"
    >
      <div
        className="MuiExpansionPanelSummary-content-36"
      >
        <h4>
          Season 6
        </h4>
      </div>
      <div
        aria-hidden="true"
        className="MuiButtonBase-root-38 MuiIconButton-root-41 MuiExpansionPanelSummary-expandIcon-37"
        onBlur={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={-1}
      >
        <span
          className="MuiIconButton-label-46"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-47"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-61"
        />
      </div>
    </div>
    <div
      aria-hidden="true"
      className="MuiCollapse-container-56"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper-58"
      >
        <div
          className="MuiCollapse-wrapperInner-59"
        >
          <div
            className="MuiExpansionPanelDetails-root-60"
          >
            <img
              className="sc-bdVaJa gENAse"
              src="https://image.tmdb.org/t/p/w500/p1udLh0gfqyZFmXBGa393gk8go5.jpg"
            />
            <div
              className="sc-htpNat gvzJoT"
            >
              <p>
                Following the shocking developments at the conclusion of season five, survivors from all parts of Westeros and Essos regroup to press forward, inexorably, towards their uncertain individual fates. Familiar faces will forge new alliances to bolster their strategic chances at survival, while new characters will emerge to challenge the balance of power in the east, west, north and south.
              </p>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Episodes This Season: 
                10
              </small>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Air Date: 
                2016-04-24
              </small>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiPaper-root-5 MuiPaper-elevation1-8 MuiPaper-rounded-6 MuiExpansionPanel-root-1 MuiExpansionPanel-rounded-2 sc-bxivhb lbdgCE"
  >
    <div
      aria-expanded={false}
      className="MuiButtonBase-root-38 MuiExpansionPanelSummary-root-32"
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex="0"
    >
      <div
        className="MuiExpansionPanelSummary-content-36"
      >
        <h4>
          Season 7
        </h4>
      </div>
      <div
        aria-hidden="true"
        className="MuiButtonBase-root-38 MuiIconButton-root-41 MuiExpansionPanelSummary-expandIcon-37"
        onBlur={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={-1}
      >
        <span
          className="MuiIconButton-label-46"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-47"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-61"
        />
      </div>
    </div>
    <div
      aria-hidden="true"
      className="MuiCollapse-container-56"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper-58"
      >
        <div
          className="MuiCollapse-wrapperInner-59"
        >
          <div
            className="MuiExpansionPanelDetails-root-60"
          >
            <img
              className="sc-bdVaJa gENAse"
              src="https://image.tmdb.org/t/p/w500/oX51n32QyHeFP5kErksemJsJljL.jpg"
            />
            <div
              className="sc-htpNat gvzJoT"
            >
              <p>
                The long winter is here. And with it comes a convergence of armies and attitudes that have been brewing for years.
              </p>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Episodes This Season: 
                7
              </small>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Air Date: 
                2017-07-16
              </small>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiPaper-root-5 MuiPaper-elevation1-8 MuiPaper-rounded-6 MuiExpansionPanel-root-1 MuiExpansionPanel-rounded-2 sc-bxivhb lbdgCE"
  >
    <div
      aria-expanded={false}
      className="MuiButtonBase-root-38 MuiExpansionPanelSummary-root-32"
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex="0"
    >
      <div
        className="MuiExpansionPanelSummary-content-36"
      >
        <h4>
          Season 8
        </h4>
      </div>
      <div
        aria-hidden="true"
        className="MuiButtonBase-root-38 MuiIconButton-root-41 MuiExpansionPanelSummary-expandIcon-37"
        onBlur={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={-1}
      >
        <span
          className="MuiIconButton-label-46"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-47"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-61"
        />
      </div>
    </div>
    <div
      aria-hidden="true"
      className="MuiCollapse-container-56"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper-58"
      >
        <div
          className="MuiCollapse-wrapperInner-59"
        >
          <div
            className="MuiExpansionPanelDetails-root-60"
          >
            <img
              className="sc-bdVaJa gENAse"
              src="https://image.tmdb.org/t/p/w500/3OcQhbrecf4F4pYss2gSirTGPvD.jpg"
            />
            <div
              className="sc-htpNat gvzJoT"
            >
              <p>
                The Great War has come, the Wall has fallen and the Night King's army of the dead marches towards Westeros. The end is here, but who will take the Iron Throne?
              </p>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Episodes This Season: 
                6
              </small>
              <small
                className="sc-bwzfXH hIxmOI"
              >
                Air Date: 
                2019-04-14
              </small>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
